### ISSUE TITLE
Esclude from Dock and app switcher

### ISSUE THREAD

**Issue by nikkouribe** [warpdotdev/Warp#4672]
### Describe the solution you'd like?

Hi, will be nice if you make that the app could be exclude from the dock and app switcher (like iterm2)

### How important is this feature to you?

4

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by ShankyJS** [comment]
@alokedesai I think this might be a duplicate of https://github.com/warpdotdev/warp/issues/1154 which should be already closed

**Comment by alokedesai** [comment]
Yeah that's right, going to mark this one as closed. Thanks for catching!
